doc/Settings.md

```diff
@@ -262,3 +263,14 @@ Experimental feature with the aim of managing dependencies, as of now it only sh
        "dependencies": true
    },
 ```
+
+### pinning
+
+This feature enables the ability to pin packages to a specific version to prevent the Windows Package Manager from updating them.
+You can enable the feature as shown below.
+
+```json
+   "experimentalFeatures": {
+       "openLogsArgument": true
```

Comment:
```suggestion
       "pinning": true
```

Reply:
Right. That's what I get for just copying the one above 😅